api/src/scout_key_censor.py

```diff
@@ -0,0 +1,43 @@
+import logging
+import threading
+from logging import LogRecord
+
+_scout_thread_state = threading.local()
+_scout_thread_state.logged_key = False
+
+
+class ScoutKeyCensor(logging.Filter):
+    """
+    Filter that stops scout_apm from logging it's API key to the logs. :(
+
+    Should be removed after this issue is resolved:
+    https://github.com/scoutapp/scout_apm_python/issues/297
```

Comment:
this issue is resolved, so we can delete this! :)

Reply:
Nice.